Jenkinsfile

```diff
@@ -1,37 +0,0 @@
-pipeline {
```

Comment:
Whats happening here

Reply:
nah

Reply:
not sure why my comment was not address here

Reply:
we are removing the file